src/pages/ReimbursementAccount/EnableStep.js

```diff
@@ -63,6 +69,7 @@ function EnableStep(props) {
                 subtitle={props.policyName}
                 shouldShowGetAssistanceButton
                 guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_BANK_ACCOUNT}
+                onBackButtonPress={() => Navigation.navigate(ROUTES.WORKSPACE_REIMBURSE.getRoute(props.policyID))}
```

Comment:
This should be handled in parent component (ReimbursementAccountPage)
```suggestion
                onBackButtonPress={props.onBackButtonPress}
```